src/vue/align-editor/align-editor.vue

```diff
@@ -1,33 +1,34 @@
 <template>
   <div class="alpheios-alignment-editor-container" v-show="showAlignEditor">
-      <h2>Define Alignment for Source and Translation Texts
+      <h2>Define Alignment for Origin and Target Texts
         (<span class="alpheios-alignment-editor-text-define-container__show-label" @click="toggleDefineAlignShow">{{ defineAlignShowLabel }}</span>)
       </h2>
       <div class="alpheios-alignment-editor-align-define-container" v-if="showAlignEditor" v-show="defineAlignShow">
-          <align-text :align-text-data="sourceText" :prefix-id = "1" @clickWord="clickWord"/>
-          <align-text :align-text-data="translationText" :prefix-id = "2" @clickWord="clickWord"/>
+          <align-editor-single-block
+            :align-text-data="originAlignedText" :prefix-id = "1" 

```

Comment:
re prefix-id - I think it's not being used anyway, but just to point out that we should not make any assumptions about how the location of the text on the display corresponds to the text identifier.  We should probably also not assume a left-to-right order has any meaning. I.e. when working with an origin text that is RTL it might make sense for the origin text to be on the right side of the display and the target on the left.

Reply:
about prefix-id:
I use it for defining tokens for origin and target texts - for now it is defined this simple way - id (1-2).
But now I believe I should retjink how to define token identification - will think about it

Reply:
I didn't finally undestand about the direction.
dir is defined for both editors - source texts and aligned texts.
Should I change it any way? could you give me some more explanations?

Reply:
> about prefix-id:
> I use it for defining tokens for origin and target texts - for now it is defined this simple way - id (1-2).
> But now I believe I should rethink how to define token identification - will think about it

We probably need to avoid using location of the token in the text as the identifier, because we need to allow the user to edit the initial tokenization results in case they are incorrect. This would require a renumbering of all the tokens. We need to keep track of the location of the tokens, but that probably needs to be a separate property from the unique identifier of the token.

Reply:
> I didn't finally undestand about the direction.
> dir is defined for both editors - source texts and aligned texts.
> Should I change it any way? could you give me some more explanations?

I just mean that we cannot assume that the origin text will be on the left of the screen and the target text will be on the right of the screen.

Reply:
about texts layout on the screen (left-to-right, right-to-left)
oh, I understood now :) will add it to the next step